week-04/README.md

```diff
@@ -13,63 +13,67 @@ The instructor will then review the pull request and make comments should furthe
 ********************************************************************************
 
 
-
 [Course Textbook Chapter 7, Problem 3]
 
+-	*Question* (aguenane−stat6250): What is the general form of the VALUE statement?
 
-
-[Course Textbook Chapter 7, Problem 4]
-
-
+ [Course Textbook Chapter 7, Problem 4]
```

Comment:
Existing content should not be edited or deleted in any way; please update PR and DM me in Slack with PR URL when ready for re-evaluation